src/solvers/LevenbergMarquardt.jl

```diff
@@ -254,7 +264,7 @@ function step_solver!(
     nlso = get_objective(dmp)
     basis_ox = _maybe_get_basis(M, lms.p, nlso.jacobian_tangent_basis)
     get_jacobian!(dmp, lms.jacF, lms.p, basis_ox)
-    λk = lms.damping_term * norm(lms.residual_values)
+    λk = lms.damping_term * norm(lms.residual_values)^2
```

Comment:
I there possibly `^2` missing here?
![image](https://github.com/JuliaManifolds/Manopt.jl/assets/6612981/5b059c27-1b37-4a10-bdd2-8d8da48ee11c)

Reply:
Yes, thanks, there should be `^2`-- sorry for the mistake.